ow_lander/urdf/lander.xacro

```diff
@@ -642,7 +649,7 @@
     <collision>
       <surface>
         <contact>
-          <collide_bitmask>0x0002</collide_bitmask>
+          <collide_bitmask>0x0001</collide_bitmask>

```

Comment:
You could just leave this block out to have collisions with terrain. It would probably be good to at least comment that this is enabling collisions with terrain.